### DISCUSSION TITLE
Auto-Py-To-Exe questions

### DISCUSSION THREAD

**Discussion by JacktheFabricRipper** [pokepetter/ursina#182] [Q&A]
So, I want to convert my project to exe, however, it says that I have to  panda3d-1.10.6.dist-info, panda3d, panda3d_tools, ursina-.1.0.dist-info, and ursina, but they don't say where I'm supposed to get these files. Does anyone have any links to these files?

https://www.ursinaengine.org/building.html

**Comment by pokepetter** [comment]
Python\Lib\site-packages\

**Reply by JacktheFabricRipper** [reply]
thank you!
